rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -467,7 +467,8 @@ object Runtime {
     def checkCreateDataDir: F[Unit] =
       for {
         notexists <- Sync[F].delay(Files.notExists(dataDir))
-        _         <- if (notexists) Sync[F].delay(Files.createDirectories(dataDir)) else ().pure[F]
+        _ <- if (notexists) Sync[F].delay(Files.createDirectories(dataDir)) >> ().pure[F]
```

Comment:
What's the point of adding `>> ().pure[F]` here?

Reply:
So that the return type of the statement is consistent and the editor will stop screaming. Prior to the change, the affirmative case returned `F[Path]` while the negative case returned `F[Unit]`. I'm actually not sure why it compiled in the first place.